docker-compose.yml

```diff
@@ -14,25 +14,52 @@ services:
     volumes:
       - postgres-db:/var/lib/postgresql/data
 
+  rabbitmq:
+    container_name: rabbitmq
+    image: rabbitmq:3.13.4-management
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    restart: always
+    environment:
+      AMQP_URL: 'amqp://rabbitmq?connection_attempts=5&retry_delay=5'
+      RABBITMQ_DEFAULT_USER: rabbitmq
```

Comment:
I know that this docker-compose file is only for development/testing purposes, but still, hard-coding passwords is a bad practice, maybe put them into the `.env` file and reference it from compose?
https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-env_file-attribute